### ISSUE TITLE
Use oxipng in optimize_images

### ISSUE THREAD

**Issue by fmarier** [fmarier/user-scripts#2]
As mentioned in [this blog post](https://seirdy.one/2020/11/23/website-best-practices.html) and used in [this script](https://git.sr.ht/~seirdy/dotfiles/tree/3b722a843f3945a1bdf98672e09786f0213ec6f6/Executables/shell-scripts/bin/optimize-image), [oxipng](https://github.com/shssoichiro/oxipng) (which is [not yet in Debian](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=913285)) is a rewrite of optipng in Rust. It focuses on improving performance, but it might also lead to better compression. To be tested.

**Comment by fmarier** [comment]
I tested this on a small image and these are the results I got:
- `original.png`: 13737 bytes
- `optipng.png`: 11191 bytes (`optipng -o9`)
- `oxipng.png`: 11240 bytes (`oxipng`)
- `oxipngmax.png`: 10658 bytes (`oxipng -o max -Z -a`)
- `smallest.png`: 10234 bytes, the result of either:
  - `optipng -o9` and `advpng -z -4`, or
  - `oxipng` and `advpng -z -4`, or
  - `oxipng -o max -Z -a` and `advpng -z -4`

In other words, by itself, it's better than `optipng`, but it's still not as good as using `optipng` and `advpng` together. There doesn't seem to be any need to swap `optipng` for `oxipng` if one uses `advpng` afterwards.
